app/component/ItineraryTab.js

```diff
@@ -217,8 +223,13 @@ class ItineraryTab extends React.Component {
     const suggestionIndex = this.context.match.params.secondHash
       ? Number(this.context.match.params.secondHash) + 1
       : Number(this.context.match.params.hash) + 1;
+
+    const co2value = typeof itinerary.emissionsPerPerson?.co2 === 'number' && itinerary.emissionsPerPerson?.co2 >= 0 ? Math.round(itinerary.emissionsPerPerson?.co2) : -1;
+    const carCo2Value = this.props.carItinerary && this.props.carItinerary.length > 0 ? Math.round(this.props.carItinerary[0].emissionsPerPerson?.co2) : 0;
+    const co2SimpleDesc =  carCo2Value === 0;
```

Comment:
this is not the description itself, maybe rename to be `useCo2SimpleDesc`, there also seems to be extra space after the `=`, weird that a linter isn't complaining about this(?).

Reply:
Fixed the name. I don't know either why linter didn't catch the gap...